libutils/json-utils.c

```diff
@@ -0,0 +1,263 @@
+#include <json-utils.h>
```

Comment:
Just for the sake of completion, in libpromises you can also include `cf3.defs.h` on top, this way you get more standard defines and it auto-includes `platform.h`.